docs/source/newsfragments/3682.bugfix.rst

```diff
@@ -0,0 +1 @@
+Pipe `test_args` through :ref:`Python Test Runner <howto-python-runner>` for Verilator
```

Comment:
```suggestion
Pass `test_args` to test command when using :ref:`Python Test Runner <howto-python-runner>` for Verilator, which was previously missing.
```
Good to mention the previous state so people can map that to a potential change in behavior.

Reply:
Also please use ` ``test_args`` ` in RST to get monospace rendering, single backticks produce italics.